lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -186,6 +188,18 @@ module Constants
                  :default => nil,
                  :secret => true
 
+    # Whether to collect metrics about the plugin usage. The way metrics are
+    # collected and exposed is controlled by the parameter monitoring_type.
```

Comment:
Being extra nit-picky... How about: `The mechanism for collecting and exposing metrics is controlled by the monitoring_type parameter.`?